fabric-fluid-extensions-v1/src/main/java/net/fabricmc/fabric/api/fluid/extension/v1/FabricFlowableFluid.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.fluid.extension.v1;
+
+import net.fabricmc.fabric.impl.fluid.extension.FlowableFluidExtensions;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.fluid.FlowableFluid;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.state.StateManager;
+import net.minecraft.state.property.IntProperty;
+
+public abstract class FabricFlowableFluid extends FlowableFluid {
+	private IntProperty stateIndexProperty;
```

Comment:
This int property is never set?

Should also be final

Reply:
It's not possible to make it final because it's set in the appendProperties method.

Reply:
OK I see.